tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -227,17 +230,29 @@ async def test_should_deploy_bytecode_at_address(
             get_starknet_address,
             get_solidity_contract,
         ):
+            nonce_initial = (
+                await plain_opcodes.contract_account.get_nonce().call()
+            ).result.nonce
+
             salt = 1234
             evm_address = await plain_opcodes.create2(
                 bytecode=counter.constructor().data_in_transaction,
                 salt=salt,
                 caller_address=plain_opcodes_deployer,
             )
-            starknet_address = get_starknet_address(salt)
+            starknet_address = get_starknet_address(int(evm_address, 16))
```

Comment:
Fixed this, we were not using the right starknet address and I was getting `StarknetErrorCode.UNINITIALIZED_CONTRACT` in the nonce assertion below